resources/js/notifications/components/NotificationsList.vue

```diff
@@ -1,167 +1,252 @@
-                <i class="fas fa-fw fa-play-circle" v-if="props.rowData.type==='TASK_CREATED'"></i>
-                <i class="fas fa-fw fa-check-circle" v-if="props.rowData.type==='TASK_COMPLETED'"></i>
-                <i class="fas fa-fw fa-user-friends" v-if="props.rowData.type==='TASK_REASSIGNED'"></i>
-                <i class="fas fa-fw fa-comment-alt" v-if="props.rowData.type==='MESSAGE'"></i>
-                <i class="fas fa-fw fa-file-download" v-if="props.rowData.type==='FILE_READY'"></i>
-                <span v-if="props.rowData.type==='FILE_READY'">
-                    <a v-bind:href="props.rowData.url">{{props.rowData.name}}</a>
-                </span>
-                <span v-else>
-                    <a v-bind:href="props.rowData.url">#{{ props.rowData.request_id }} {{props.rowData.name}}</a>
-                    ({{props.rowData.processName}})
-                </span>
-            </template>
-
-            <template slot="changeStatus" slot-scope="props">
-                <span v-if="props.rowData.read_at === null" style="cursor:pointer" @click="read(props.rowData.id)"
-                      class="far fa-envelope fa-lg">
-                </span>
-
-                <span v-if="props.rowData.read_at !==  null" style="cursor:pointer" @click="unread(props.rowData.id)">
-                   <i class="far fa-envelope-open fa-lg"></i>
-                </span>
-            </template>
-
-        </vuetable>
-        <pagination :single="$t('Task')" :plural="$t('Tasks')" :perPageSelectEnabled="true" @changePerPage="changePerPage"
-                    @vuetable-pagination:change-page="onPageChange" ref="pagination"></pagination>
-        </div>
+  <div class="data-table">
+    <div
+      class="card card-body table-card"
+    >
+      <vuetable
+        :data-manager="dataManager"
+        :sort-order="sortOrder"
+        :css="css"
+        :api-mode="false"
+        :fields="fields"
+        :data="data"
+        data-path="data"
+        pagination-path="meta"
+        :no-data-template="$t('No Data Available')"
+      >
+        <template
+          slot="changeStatus"
+          slot-scope="props"
+        >
+          <span
+            v-if="props.rowData.read_at === null"
+            style="cursor:pointer"
+            class="far fa-envelope fa-lg blue-envelope"
+            @click="read(props.rowData.id)"
+          />
+          <span
+            v-if="props.rowData.read_at !== null"
+            style="cursor:pointer"
+            @click="unread(props.rowData.id)"
+          >
+            <i class="far fa-envelope-open fa-lg gray-envelope" />
+          </span>
+        </template>
+
+        <template
+          slot="from"
+          slot-scope="props"
+        >
+          <notification-user
+            :notification="props.rowData"
+          />
+        </template>
+
+        <template
+          slot="subject"
+          slot-scope="props"
+        >
+          <a :href="props.rowData.url">
```

Comment:
@pmoskrojas Can you change this to a `@click` event and `cursor: pointer;` like resources/js/notifications/components/notification-item.vue does so we don't change the color?

Also, comments are not linking for some reason but they do link in the bell icon popover